### ISSUE TITLE
Dockerimage node:slim can't get ssh://git@github.com/fabien88/simple-text-parser.git

### ISSUE THREAD

**Issue by de-dan** [fabien88/ssmd#7]
Hi,

now that you are using a none-npm-listed node module (simple-text-parser), the dockerimage `node:slim` fails while installing the dependencies. `node:slim` doesn't have git nor ssh installed.

I can have a workaround with `node:alpine` + installing some additional packages, but... is your fork of simple-text-parser really necessary?

```
npm ERR! code ENOENT
npm ERR! syscall spawn git
npm ERR! path git
npm ERR! errno -2
npm ERR! enoent Error while executing:
npm ERR! enoent undefined ls-remote -h -t ssh://git@github.com/fabien88/simple-text-parser.git
npm ERR! enoent 
npm ERR! enoent 
npm ERR! enoent spawn git ENOENT
npm ERR! enoent This is related to npm not being able to find a file.
npm ERR! enoent 
```

**Comment by fabien88** [comment]
ah ! Ok I've published the fork on npm. Should works now with the released version 1.0.7

**Comment by de-dan** [comment]
Thank you!
